src/store/features/board/thunks.ts

```diff
@@ -307,3 +307,8 @@ export const importBoard = createAsyncThunk<void, string, {state: ApplicationSta
     "deleteBoard"
   ).then((boardID) => window.location.assign(`/board/${boardID}`));
 });
+
+// helper function to handle board deletion redirects
+const redirectToBoardDeletedPage = () => {
+  window.location.replace("/?boardDeleted=true");
+};
```

Comment:
please fix the linter warning (used before definition)

I recommend running `yarn run lint` and `yarn run test` to see if everything works before pushing :)